projects/plugins/crm/modules/woo-sync/admin/settings/main.page.php

```diff
@@ -305,7 +305,12 @@ function jpcrm_settings_page_html_woosync_main() {
 						<td class="wfieldname" colspan="2">
 							<label><?php esc_html_e( 'Order status map', 'zero-bs-crm' ); ?>:</label><br />
 							<?php esc_html_e( 'Here you can choose how you want to map WooCommerce order statuses to CRM statuses (if the above setting is enabled)', 'zero-bs-crm' ); ?>
-							<br/>
+							<div style="margin-top:10px;">
+								<a style="margin-top:10px;" href="<?php echo esc_url( $zbs->urls['woomanagingorders'] ); ?>" target="_blank">
+									<?php esc_html_e( 'Learn more about the WooCommerce order statuses', 'zero-bs-crm' ); ?>
+									<img class="jpcrm-external-link-icon" style="width:1em" src="<?php echo esc_url( ZEROBSCRM_URL ); ?>i/external-link.svg" />
```

Comment:
I notice that the external link icon is sitting slightly higher than it does within the popup when clicking on the info icon (eg. on the Dashboard / Contact pages). 

On the WooSync page:
<img width="73" alt="Screenshot 2023-05-11 at 17 04 59" src="https://github.com/Automattic/jetpack/assets/16754605/39aec512-843b-46ab-8c3a-522e9631d794">

In the 'learn more' popup:
<img width="72" alt="Screenshot 2023-05-11 at 17 05 57" src="https://github.com/Automattic/jetpack/assets/16754605/bc8787ec-4ffb-4c20-b09f-9bfa58e9c8b8">


That's likely because the `jpcrm-external-link-icon` class is located within a learn menu specific scss file: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/crm/sass/_JetpackCRM.admin.learnmenu.scss#L81

Seems like it needs to go in a more central location so it can be picked up elsewhere. 

Absolutely not a blocker for this PR, I can follow up with that separately.

Reply:
>  Nice idea with the link.

Credits for Thomas. He suggested it at the last minute :)